README.md

```diff
@@ -182,16 +183,16 @@ Just do some preparation work, including:
     - **Day 20:** Pre & Task 6
 
 - For students in SEIEE CS:
-  - You are supposed to finish `Task0`, `Task1 (40pts)`, `Task2 (40pts)`, `Task5 (10pts)` and `Task6 (10pts)`.
-  - Recommended schedule:
+  - You are supposed to finish `Task0`, `Task1 (40pts)`, `Task2 (40pts)`, `Task5 (10pts)` and `Task6 (10pts)`
+  - Recommended timeline:
     - **Day 1:**  Task 0: Environment setup
     - **Day 2 - 3:** Task 0: Learn Rust
     - **Day 4 - 6:** Task 1 & 5
     - **Day 7 - 9:** Task 2 & 5
     - **Day 10:** Pre & Task 6
 
-### Others
+### More Tips
 
-- You should have a document recording all the bonus you have done. For each bonus, you should show the code or the effect of your work.
-- There is no restriction on unsafe code, if you think it is necessary, you can use it, and explain why.
-- There will be 2 students to do presentations for ACM Class and SEIEE CS. They will get extra points based on their presentations, no more than 10 points.
+- You should have a document recording all the bonus you have done. For each bonus, you should show the code or a benchmark of your work
```

Comment:
The list is comprised of full sentences. Full sentences should be ended by full periods.

Reply:
Ok